src/types/custom.d.ts

```diff
@@ -37,11 +33,11 @@ declare module '*.jpeg' {
 // NOTE: this will not catch keys/classnames that do not exist in the stylesheet
 declare module '*.module.scss' {
   const classes: { [key: string]: string };
-  export default classes;
+  export = classes;
```

Comment:
Big changes to css-loader (see webpack config changes) regarding CSS modules and corresponding types.
It now exposes classes as named exports instead of all on the default export, which means by default we should change imports to `import * as styles from...` to keep things working, but I was having issues with types regardless. There is an option to keep the current behaviour, but it's not default so maybe in the future it goes away...
See webpack config

Reply:
Not sure I understand, at present nothing needs to change with regards to style importing but may need to in the future?

Reply:
Yes, no need to change usage now, but I did have to change types declaration and add more config to make it work in the way we are currently using it.